Bottom: 1abf3ed14079a13eb818af5190a106dee248db45
Top:    f164593920b4a57f062c4d4e8cc1a4ff88eaaf9c
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-15 17:11:07 -0500

Refresh of use-docker-named-networks

---
diff --git a/roles/docker-exim4/tasks/main.yml b/roles/docker-exim4/tasks/main.yml
index eb37e6a6..e71c619f 100644
--- a/roles/docker-exim4/tasks/main.yml
+++ b/roles/docker-exim4/tasks/main.yml
@@ -109,6 +109,8 @@
     image: '{{ exim4_image_name }}:{{ docker_image_tag }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+        aliases:
+          - '{{ exim4_hostname }}'
     ports: '{{ exim4_port_args }}'
     purge_networks: true
     restart_policy: '{{ docker_restart_policy }}'
diff --git a/roles/docker-mantisbt/tasks/main.yml b/roles/docker-mantisbt/tasks/main.yml
index 5752ba3b..a9404bc6 100644
--- a/roles/docker-mantisbt/tasks/main.yml
+++ b/roles/docker-mantisbt/tasks/main.yml
@@ -174,10 +174,11 @@
     name: '{{ mantisbt_container_name }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+        aliases:
+          - '{{ mantisbt_hostname }}'
         links:
           - '{{ openldap_container_name }}:mantisbt_ldap'
           - '{{ mantisbt_db_container_name }}:mantisbt_db'
-          - '{{ exim4_container_name }}:{{ exim4_hostname }}'
     ports: '{{ mantisbt_port_args }}'
     purge_networks: true
     restart_policy: '{{ docker_restart_policy }}'
diff --git a/roles/docker-mediawiki/tasks/main.yml b/roles/docker-mediawiki/tasks/main.yml
index 3a5542c5..6fa64e9d 100644
--- a/roles/docker-mediawiki/tasks/main.yml
+++ b/roles/docker-mediawiki/tasks/main.yml
@@ -147,10 +147,11 @@
     name: '{{ wiki_container_name }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+        aliases:
+          - '{{ wiki_hostname }}'
         links:
           - '{{ openldap_container_name }}:wiki_ldap'
           - '{{ wiki_db_container_name }}:wiki_db'
-          - '{{ exim4_container_name }}:{{ exim4_hostname }}'
     ports: '{{ wiki_port_args }}'
     purge_networks: true
     restart_policy: '{{ docker_restart_policy }}'
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index 5422d3e5..ba3177ff 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -116,6 +116,8 @@
     name: '{{ openldap_container_name }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+        aliases:
+          - '{{ openldap_hostname }}'
     ports: '{{ openldap_port_args }}'
     purge_networks: true
     restart_policy: '{{ docker_restart_policy }}'
diff --git a/roles/docker-svn/tasks/main.yml b/roles/docker-svn/tasks/main.yml
index 49952df6..a76dfae6 100644
--- a/roles/docker-svn/tasks/main.yml
+++ b/roles/docker-svn/tasks/main.yml
@@ -83,11 +83,11 @@
       SVN_LDAP_PASSWORD: '{{ openldap_proxyagent_password }}'
     hostname: '{{ svn_hostname }}'
     image: '{{ svn_image_name }}:{{ docker_image_tag }}'
-    link:
-      - '{{ openldap_container_name }}:ldap'
     name: '{{ svn_container_name }}'
     networks:
       - name: '{{ docker_network_frontend }}'
+        aliases:
+          - '{{ svn_hostname }}'
     ports: '{{ svn_port_args }}'
     purge_networks: true
     restart_policy: '{{ docker_restart_policy }}'
